src/core/FFmpeg.js

```diff
@@ -124,7 +124,7 @@ class FFmpeg extends Duplex {
         // Do nothing
       }
     }
-    throw new Error('FFmpeg not found!');
+    throw new Error('FFmpeg/Avconv not found!');
```

Comment:
```suggestion
    throw new Error('FFmpeg/avconv not found!');
```

See the [official documentation](https://libav.org/avconv.html), `avconv` is the official name.